src/Orchard/ContentManagement/DefaultHqlQuery.cs

```diff
@@ -216,8 +220,7 @@ public class DefaultHqlQuery : IHqlQuery {
 
         public string ToSql(bool count) {
             var sessionImp = (ISessionImplementor)_session;
-            var translatorFactory = new ASTQueryTranslatorFactory();
-            var translators = translatorFactory.CreateQueryTranslators(ToHql(count), null, false, sessionImp.EnabledFilters, sessionImp.Factory);
+            var translators = TranslatorFactory.CreateQueryTranslators(ToHql(count), null, false, sessionImp.EnabledFilters, sessionImp.Factory);
```

Comment:
I checked here: https://github.com/nhibernate/nhibernate-core/blob/master/src/NHibernate/Hql/Ast/ANTLR/ASTQueryTranslatorFactory.cs#L27

And it seems you can use the static method directly, no need to instantiate anything.